recipe/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set name = "intel-extension-for-pytorch" %}
+{% set version = "1.12.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/intel/{{ name }}/archive/refs/tags/v{{ version }}00.tar.gz
+  sha256: 6f2b54fc3e0a5e26ccaa0552ba93e6d7ecbdc2902b4920e86fde5efdfe427ebc
+  patches:
+    - patches/0001-add-git-rev.patch
+
+build:
+  number: 0
+  # Anything other than linux-64 is not supported fully by upstream, and 3.11
+  # doesn't have PyTorch yet.
+  skip: True  # [py<38 or py>310]
+  skip: True  # [not linux64]
+  missing_dso_whitelist:
+    - "$RPATH/libc10.so"
+    - "$RPATH/libLLVM-14.so"
+    - "$RPATH/libtorch_cpu.so"
+    - "$RPATH/libtorch_python.so"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - llvmdev 14.0.6
+    - cmake 3.22.1
+    - patch
+    - make
+    - git                     # [not win]
+  host:
+    - python
+    - pip
+    - wheel
+    - setuptools
+    - packaging 23.0
+    - pybind11 2.10.1
+    - pytorch {{ version }}
+  run:
+    - python
+    - psutil
+    - torchvision
+    # Both OpenMP installations can be used with this package (can be selected at runtime).
+    - intel-openmp
+    - _openmp_mutex
+    - libllvm14 14.0.6
+    - pytorch {{ version }}
+
+test:
+  requires:
+    - pip
+    - torchvision
+  commands:
+    - pip check
+    - python -c "import torch; import torchvision; import intel_extension_for_pytorch"
```

Comment:
I would modified it as:
`python -c "import torch; import torchvision; import intel_extension_for_pytorch as ipex; print('torch:', torch.__version__,' ipex:',ipex.__version__)"`
see https://intel.github.io/intel-extension-for-pytorch/latest/tutorials/installation.html#build-docker-container-from-dockerfile

Reply:
Thanks.